packages/embed/src/subjects.ts

```diff
@@ -1,30 +1,37 @@
 import { createSubject } from './utils/create-subject'
 
-export const approvalRequired$ = createSubject<boolean>()
-export const approvalUrl$ = createSubject<string>()
-export const approvalStarted$ = createSubject()
-export const approvalCancelled$ = createSubject()
-export const approvalLost$ = createSubject()
-export const approvalCompleted$ = createSubject()
-export const frameHeight$ = createSubject<number>(0)
-export const optionsLoaded$ = createSubject<boolean>(false)
-export const formSubmit$ = createSubject()
-export const transactionCreated$ = createSubject<{
-  id: string
-  status: string
-  paymentMethod?: { id?: string }
-}>()
-export const transactionFailed$ = createSubject()
-
-// Initial events
-formSubmit$.subscribe(() => {
-  if (approvalRequired$.value()) {
-    approvalStarted$.next()
+export const createSubjectManager = () => {
```

Comment:
This was one of the main issues - these were kept between instances before, but are now instantiated each time `setup` is called.